content/docs/introduction/roadmap.md

```diff
@@ -38,16 +45,22 @@ GitHub issue: [#398](https://github.com/prometheus/prometheus/issues/398)
 At this time, metric types and other metadata are only used in the
 client libraries and in the exposition format, but not persisted or
 utilized in the Prometheus server. We plan on making use of this
-metadata in the future. For example, we could suggest automatic rates
-over counters, warn users if they take the rate of a gauge, or display
-metric documentation strings. Some metric types, like the upcoming
-[server-side histograms](https://github.com/prometheus/prometheus/issues/480),
-could also be stored and processed in a more efficient way.  The
-details of this are still to be determined.
+metadata in the future. The first step is to aggregate this data in-memory
+in Prometheus and provide it via an experimantal API endpoint.
```

Comment:
As I explicitly stated in a recent discussion, my understanding was that complex sample types to store histograms more efficiently are not off the table. It is also a problem that is relevant in various way for a typical major Prometheus user, as I can witness it in my day-to-day work.

My objection is completely ignored by this PR.